django/contrib/auth/hashers.py

```diff
@@ -322,53 +323,76 @@ def encode(self, password, salt):
             parallelism=self.parallelism,
             hash_len=argon2.DEFAULT_HASH_LENGTH,
             type=argon2.low_level.Type.I,
+            version=self.version,
```

Comment:
well, you can drop version in `encode` and just use `argon2.low_level.ARGON2_VERSION` in `must_update` directly -- loading there is fine after all (or do we have another occurance of it?)